jx_sqlite/expressions.py

```diff
@@ -117,14 +117,22 @@ def to_sql(self, schema, not_null=False, boolean=False):
         Log.error("Can only handle Variable")
     term = self.term.var
     prefix_length = len(split_field(term))
-    return wrap([
-        {
-            "name": join_field(split_field(schema.get_column_name(c))[prefix_length:]),
-            "sql": Variable(schema.get_column_name(c)).to_sql(schema)[0].sql
-        }
-        for n, cols in schema.map_to_sql(term).items()
-        for c in cols
-    ])
+    db_columns = []
+    for n, cols in schema.map_to_sql(term).items():
+        for c in cols:
```

Comment:
I am wondering if this code, and this double loop, is old code:  Can we use `schema.leaves()` instead?

Reply:
Is there a `leaves()` function on `Schema` objects?

Reply:
maybe I was thinking of the ES version of this project:

https://github.com/klahnakoski/ActiveData/blob/b9f1d3193c5e56965dc7b4d219a5288a6a06ffaa/jx_base/schema.py#L73

which will be added to jx-sqlite sometime in the future